uaclient/messages/__init__.py

```diff
@@ -1534,13 +1534,14 @@ def __repr__(self):
 the --access-only flag.""",
 )
 
-UNEXPECTED_ERROR = NamedMessage(
+UNEXPECTED_ERROR = FormattedNamedMessage(
     "unexpected-error",
     t.gettext(
         """\
-Unexpected error(s) occurred.
-For more details, see the log: /var/log/ubuntu-advantage.log
-To file a bug run: ubuntu-bug ubuntu-advantage-tools"""
+Unexpected error(s) occurred: {error_msg}
+For more details, see the log: {log_path}
+First, try searching online for solutions to this error.
```

Comment:
But if your point is that we could have more.. ahem, sensible wording, then yes we could

Reply:
Brb gonna go run this through chatGPT to make it more friendly.

Reply:
It could be as simple as adding an "if", e.g. "If you think this is a bug, please run: ubuntu-bug ubuntu-advantage-tools"

Reply:
Putting this all together. How does the following sound?
```
Consider searching online for solutions to this error.
Or, if you think this is a bug, please run: ubuntu-bug ubuntu-advantage-tools
```

Reply:
I meant that I think we don't need to explicitly ask the user to search online at all (even though I'm the one who suggested this). Seeing it written out feels a little too much like we're pushing too much responsibility on the user.
I do still think that the wording "To file a bug run: ubuntu-bug ubuntu-advantage-tools" or similar is too imperative though. So my idea is to only add a conditional ("if") to that suggestion.
I think the "if" suggests enough that it isn't necessarily a bug and could be worth searching online for.
So I think in total it would be:
```
Unexpected error(s) occurred: {error_msg}
For more details, see the log: {log_path}
If you think this is a bug, please run: ubuntu-bug ubuntu-advantage-tools
```

Reply:
Thank you! And sorry for flip-flopping on that idea :upside_down_face:

Reply:
No worries! Thank you for re-evaluating this change's requirements :)